### DISCUSSION TITLE
Manual Update

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#84] [Q&A]
Having some issues with updating to the latest version because of certain hosts.  Can you tell me what folders need to be manually updated to upgrade to the latest version?

Thanks!

**Comment by PeterBowater** [comment]
Simplest way to do this is to create a new installation of Masa, then copy your site files/theme and settings.ini.cfm over to the new installation.

**Comment by guustnieuwenhuis** [comment]
This is the list of folders and files to update manually:

```
admin
config
core
Application.cfc
box.json
COPYING
gulpfile.js
htaccess.txt
index.cfm
Installation.txt
license.txt
package-lock.json
package.json
README.md
robots.txt
web.config.txt
```
